.github/workflows/sync-base-master.yaml

```
@@ -0,0 +1,60 @@
+name: Sync base-master from base/account-sdk
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    # Runs every midnight. Cron is in UTC.
+    - cron: "0 0 * * *"
+
+permissions:
+  contents: write  # required to push back to the repo
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout this repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # we need full history to update branches cleanly
+
+      - name: Configure git user
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Is the prefix necessary?


```suggestion
          git config user.email github-actions[bot]@users.noreply.github.com
```

---

To my understanding, yes it is. It's an user id for the bot: 
https://github.com/actions/checkout?tab=readme-ov-file#push-a-commit-using-the-built-in-token